src/_includes/config/setup-cron_2.4_about.md

```diff
@@ -0,0 +1,12 @@
+### About the Magento crontab
+
+The Magento _crontab_ is the configuration used to run Magento cron jobs.
+
+Magento uses cron tasks that can run with different configurations:
+
+*  PHP command-line configuration: The general cron job that reindexes indexers, generates e-mails, generates the sitemap, and so on.
```

Comment:
I removed the second bullet from the original list because it appears to be specific to the Web Setup Wizard and therefore no longer necessary:

> Web server PHP plug-in configuration: Two other cron jobs are used by the [Component Manager and System Upgrade utilities](https://devdocs.magento.com/guides/v2.3/comp-mgr/bk-compman-upgrade-guide.html).

Perhaps a better solution is to remove the remaining bullet and combine it with the lead-in sentence.